frontend/packages/db-structure/src/parser/schemarb/parser.ts

```diff
@@ -135,6 +158,14 @@ function extractTableDetails(blockNodes: Node[]): [Column[], Index[]] {
           if (node.name === 'index') {
             const index = extractIndexDetails(node)
             indexes.push(index)
+            if (index.unique && index.columns[0]) {
+              const uniqueConstraint: UniqueConstraint = {
+                type: 'UNIQUE',
+                name: `UNIQUE_${index.columns[0]}`,
+                columnName: index.columns[0],
```

Comment:
Just a quick question for clarification!
Are we extracting **only the first item** from the columns array here because of the reasoning mentioned in this comment?

https://github.com/liam-hq/liam/pull/1168#discussion_r2032309347

Reply:
> extracting only the first item from the columns array here because of the reasoning mentioned in this comment

Yes, and same for the other parsers as well (e.g. this is an implementation of [tbls's Unique constraints](https://github.com/liam-hq/liam/blob/76f945446fd2e6e3c3cf58c57fecbdec8e4e34bf/frontend/packages/db-structure/src/parser/tbls/parser.ts#L167-L177))